.github/workflows/test.yml

```diff
@@ -39,8 +40,7 @@ jobs:
 
       - name: Install stylua
         run: |
-          URL=$(curl -L https://api.github.com/repos/JohnnyMorganz/StyLua/releases/latest | jq -r '.assets[] | select(.name == "stylua-linux-x86_64.zip") | .browser_download_url')
-          wget --directory-prefix="$BIN_DIR" "$URL"
+          wget --directory-prefix="$BIN_DIR" "https://github.com/JohnnyMorganz/StyLua/releases/download/${STYLUA_VERSION}/stylua-linux-x86_64.zip"
```

Comment:
https://github.com/JohnnyMorganz/StyLua/releases/download/latest/stylua-linux-x86_64.zip works for me